### ISSUE TITLE
failing actix benches on techempower

### ISSUE THREAD

**Issue by cdbattags** [actix/actix-web#1403]
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix-core
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix-diesel
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix-pg
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix-pg
- https://tfb-status.techempower.com/unzip/results.2020-03-08-16-57-15-762.zip/actix-raw

```
actix: error: could not compile `failure_derive`.
```

Obviously this repo is not the best place for this issue but it would be worth it if someone had the spare time to go fix:

https://github.com/TechEmpower/FrameworkBenchmarks/tree/master/frameworks/Rust/actix

---

Full log:

```
actix: Step 1/7 : FROM rust:1.41
actix:  ---> a716da47d5b9
actix: Step 2/7 : RUN apt update -yqq && apt install -yqq cmake g++
actix:  ---> Running in 77f72dd95299
actix: 
actix: WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
actix: All packages are up to date.
actix: 
actix: WARNING: apt does not have a stable CLI interface. Use with caution in scripts.
actix: g++ is already the newest version (4:8.3.0-1).
actix: The following additional packages will be installed:
actix:   cmake-data libarchive13 libjsoncpp1 librhash0 libuv1
actix: Suggested packages:
actix:   cmake-doc ninja-build lrzip
actix: The following NEW packages will be installed:
actix:   cmake cmake-data libarchive13 libjsoncpp1 librhash0 libuv1
actix: debconf: delaying package configuration, since apt-utils is not installed
actix: 0 upgraded, 6 newly installed, 0 to remove and 0 not upgraded.
actix: Need to get 5578 kB of archives.
actix: After this operation, 26.9 MB of additional disk space will be used.
actix: Selecting previously unselected package cmake-data.
actix: (Reading database ... 
actix: (Reading database ... 5%
actix: (Reading database ... 10%
actix: (Reading database ... 15%
actix: (Reading database ... 20%
actix: (Reading database ... 25%
actix: (Reading database ... 30%
actix: (Reading database ... 35%
actix: (Reading database ... 40%
actix: (Reading database ... 45%
actix: (Reading database ... 50%
actix: (Reading database ... 55%
actix: (Reading database ... 60%
actix: (Reading database ... 65%
actix: (Reading database ... 70%
actix: (Reading database ... 75%
actix: (Reading database ... 80%
actix: (Reading database ... 85%
actix: (Reading database ... 90%
actix: (Reading database ... 95%
actix: (Reading database ... 100%
actix: (Reading database ... 23968 files and directories currently installed.)
actix: Preparing to unpack .../0-cmake-data_3.13.4-1_all.deb ...
actix: Unpacking cmake-data (3.13.4-1) ...
actix: Selecting previously unselected package libarchive13:amd64.
actix: Preparing to unpack .../1-libarchive13_3.3.3-4+deb10u1_amd64.deb ...
actix: Unpacking libarchive13:amd64 (3.3.3-4+deb10u1) ...
actix: Selecting previously unselected package libjsoncpp1:amd64.
actix: Preparing to unpack .../2-libjsoncpp1_1.7.4-3_amd64.deb ...
actix: Unpacking libjsoncpp1:amd64 (1.7.4-3) ...
actix: Selecting previously unselected package librhash0:amd64.
actix: Preparing to unpack .../3-librhash0_1.3.8-1_amd64.deb ...
actix: Unpacking librhash0:amd64 (1.3.8-1) ...
actix: Selecting previously unselected package libuv1:amd64.
actix: Preparing to unpack .../4-libuv1_1.24.1-1_amd64.deb ...
actix: Unpacking libuv1:amd64 (1.24.1-1) ...
actix: Selecting previously unselected package cmake.
actix: Preparing to unpack .../5-cmake_3.13.4-1_amd64.deb ...
actix: Unpacking cmake (3.13.4-1) ...
actix: Setting up libarchive13:amd64 (3.3.3-4+deb10u1) ...
actix: Setting up libuv1:amd64 (1.24.1-1) ...
actix: Setting up librhash0:amd64 (1.3.8-1) ...
actix: Setting up cmake-data (3.13.4-1) ...
actix: Setting up libjsoncpp1:amd64 (1.7.4-3) ...
actix: Setting up cmake (3.13.4-1) ...
actix: Processing triggers for libc-bin (2.28-10) ...
actix: Removing intermediate container 77f72dd95299
actix:  ---> 04bab6928e15
actix: Step 3/7 : ADD ./ /actix
actix:  ---> 7925d1c2a8da
actix: Step 4/7 : WORKDIR /actix
actix:  ---> Running in 1d9d884cc4f8
actix: Removing intermediate container 1d9d884cc4f8
actix:  ---> 3becafa28035
actix: Step 5/7 : RUN cargo clean
actix:  ---> Running in 87a0452c58f5
actix: Removing intermediate container 87a0452c58f5
actix:  ---> 63cb06194c13
actix: Step 6/7 : RUN RUSTFLAGS="-C target-cpu=native" cargo build --release
actix:  ---> Running in 7a59ded83af5
actix:     Updating crates.io index
actix:     Updating git repository `https://github.com/fafhrd91/rust-postgres.git`
actix:  Downloading crates ...
actix:   Downloaded md5 v0.7.0
actix:   Downloaded hmac v0.7.1
actix:   Downloaded env_logger v0.7.1
actix:   Downloaded memchr v2.3.3
actix:   Downloaded serde_derive v1.0.104
actix:   Downloaded base64 v0.11.0
actix:   Downloaded serde v1.0.104
actix:   Downloaded byteorder v1.3.4
actix:   Downloaded actix-rt v1.0.0
actix:   Downloaded percent-encoding v2.1.0
actix:   Downloaded sha2 v0.8.1
actix:   Downloaded fallible-iterator v0.2.0
actix:   Downloaded log v0.4.8
actix:   Downloaded serde_json v1.0.48
actix:   Downloaded url v1.7.2
actix:   Downloaded rand v0.7.3
actix:   Downloaded markup v0.4.1
actix:   Downloaded http v0.2.0
actix:   Downloaded stringprep v0.1.2
actix:   Downloaded num_cpus v1.12.0
actix:   Downloaded futures v0.3.4
actix:   Downloaded actix-server v1.0.2
actix:   Downloaded pin-project-lite v0.1.4
actix:   Downloaded bytes v0.5.4
actix:   Downloaded jemallocator v0.3.2
actix:   Downloaded tokio v0.2.6
actix:   Downloaded v_htmlescape v0.4.5
actix:   Downloaded diesel v1.4.3
actix:   Downloaded snmalloc-rs v0.2.5
actix:   Downloaded tokio-util v0.2.0
actix:   Downloaded askama v0.8.0
actix:   Downloaded yarte v0.3.5
actix:   Downloaded actix-codec v0.2.0
actix:   Downloaded actix-service v1.0.5
actix:   Downloaded actix-http v1.0.1
actix:   Downloaded phf v0.8.0
actix:   Downloaded actix-utils v1.0.6
actix:   Downloaded actix-web v2.0.0
actix:   Downloaded actix v0.9.0
actix:   Downloaded mio-uds v0.6.7
actix:   Downloaded chrono v0.4.10
actix:   Downloaded slab v0.4.2
actix:   Downloaded iovec v0.1.4
actix:   Downloaded rand_core v0.5.1
actix:   Downloaded indexmap v1.3.2
actix:   Downloaded fnv v1.0.6
actix:   Downloaded opaque-debug v0.2.3
actix:   Downloaded atty v0.2.14
actix:   Downloaded encoding_rs v0.8.22
actix:   Downloaded yarte_helpers v0.3.5
actix:   Downloaded sha1 v0.6.0
actix:   Downloaded copyless v0.1.4
actix:   Downloaded libc v0.2.67
actix:   Downloaded futures-io v0.3.4
actix:   Downloaded signal-hook-registry v1.2.0
actix:   Downloaded cfg-if v0.1.10
actix:   Downloaded syn v1.0.16
actix:   Downloaded proc-macro2 v1.0.9
actix:   Downloaded serde_urlencoded v0.6.1
actix:   Downloaded net2 v0.2.33
actix:   Downloaded rand_chacha v0.2.1
actix:   Downloaded matches v0.1.8
actix:   Downloaded unicode-bidi v0.3.4
actix:   Downloaded idna v0.1.5
actix:   Downloaded askama_escape v0.2.0
actix:   Downloaded actix-connect v1.0.2
actix:   Downloaded rand_pcg v0.2.1
actix:   Downloaded actix-threadpool v0.3.1
actix:   Downloaded futures-task v0.3.4
actix:   Downloaded pin-project v0.4.8
actix:   Downloaded yarte_derive v0.3.5
actix:   Downloaded actix-macros v0.1.1
actix:   Downloaded bitflags v1.2.1
actix:   Downloaded lazy_static v1.4.0
actix:   Downloaded getrandom v0.1.14
actix:   Downloaded futures-channel v0.3.4
actix:   Downloaded either v1.5.3
actix:   Downloaded mime v0.3.16
actix:   Downloaded httparse v1.3.4
actix:   Downloaded block-buffer v0.7.3
actix:   Downloaded askama_derive v0.8.0
actix:   Downloaded derive_more v0.99.3
actix:   Downloaded h2 v0.2.2
actix:   Downloaded language-tags v0.2.2
actix:   Downloaded futures-core v0.3.4
actix:   Downloaded fxhash v0.2.1
actix:   Downloaded markup-proc-macro v0.4.1
actix:   Downloaded futures-util v0.3.4
actix:   Downloaded yarte_config v0.3.5
actix:   Downloaded fake-simd v0.1.2
actix:   Downloaded unicode-normalization v0.1.12
actix:   Downloaded digest v0.8.1
actix:   Downloaded futures-executor v0.3.4
actix:   Downloaded crypto-mac v0.7.0
actix:   Downloaded pq-sys v0.4.6
actix:   Downloaded itoa v0.4.5
actix:   Downloaded time v0.1.42
actix:   Downloaded futures-sink v0.3.4
actix:   Downloaded termcolor v1.1.0
actix:   Downloaded encoding v0.2.33
actix:   Downloaded humantime v1.3.0
actix:   Downloaded percent-encoding v1.0.1
actix:   Downloaded ryu v1.0.2
actix:   Downloaded regex v1.3.4
actix:   Downloaded v_escape v0.7.4
actix:   Downloaded askama_shared v0.8.0
actix:   Downloaded mio v0.6.21
actix:   Downloaded diesel_derives v1.4.1
actix:   Downloaded quote v1.0.3
actix:   Downloaded snmalloc-sys v0.2.5
actix:   Downloaded jemalloc-sys v0.3.2
actix:   Downloaded smallvec v1.2.0
actix:   Downloaded toml v0.5.6
actix:   Downloaded byte-tools v0.3.1
actix:   Downloaded proc-macro2 v0.4.30
actix:   Downloaded proc-macro-hack v0.5.11
actix:   Downloaded block-padding v0.1.5
actix:   Downloaded actix-web-codegen v0.2.1
actix:   Downloaded arc-swap v0.4.4
actix:   Downloaded threadpool v1.7.1
actix:   Downloaded quote v0.6.13
actix:   Downloaded nom v4.2.3
actix:   Downloaded encoding-index-singlebyte v1.20141219.5
actix:   Downloaded unicode-xid v0.2.0
actix:   Downloaded dtoa v0.4.5
actix:   Downloaded c2-chacha v0.2.3
actix:   Downloaded pin-project-internal v0.4.8
actix:   Downloaded generic-array v0.12.3
actix:   Downloaded proc-macro-nested v0.1.3
actix:   Downloaded humansize v1.1.0
actix:   Downloaded trust-dns-resolver v0.18.0-alpha.2
actix:   Downloaded encoding-index-tradchinese v1.20141219.5
actix:   Downloaded awc v1.0.1
actix:   Downloaded pin-utils v0.1.0-alpha.4
actix:   Downloaded encoding-index-japanese v1.20141219.5
actix:   Downloaded actix-testing v1.0.0
actix:   Downloaded encoding-index-korean v1.20141219.5
actix:   Downloaded encoding-index-simpchinese v1.20141219.5
actix:   Downloaded futures-macro v0.3.4
actix:   Downloaded prettyprint v0.7.0
actix:   Downloaded toml v0.4.10
actix:   Downloaded crossbeam-channel v0.4.2
actix:   Downloaded num-traits v0.2.11
actix:   Downloaded aho-corasick v0.7.9
actix:   Downloaded regex-syntax v0.6.16
actix:   Downloaded parking_lot v0.10.0
actix:   Downloaded mime_guess v2.0.3
actix:   Downloaded url v2.1.1
actix:   Downloaded actix-tls v1.0.0
actix:   Downloaded syn v0.15.44
actix:   Downloaded actix_derive v0.5.0
actix:   Downloaded autocfg v1.0.0
actix:   Downloaded thread_local v1.0.1
actix:   Downloaded quick-error v1.2.3
actix:   Downloaded v_escape_derive v0.5.6
actix:   Downloaded phf_shared v0.8.0
actix:   Downloaded subtle v1.0.0
actix:   Downloaded num-integer v0.1.42
actix:   Downloaded tempfile v3.1.0
actix:   Downloaded toolchain_find v0.1.4
actix:   Downloaded trust-dns-proto v0.18.0-alpha.2
actix:   Downloaded actix-router v0.2.4
actix:   Downloaded v_eval v0.1.0
actix:   Downloaded resolv-conf v0.6.2
actix:   Downloaded version_check v0.1.5
actix:   Downloaded unicode-xid v0.1.0
actix:   Downloaded ansi_term v0.11.0
actix:   Downloaded failure v0.1.6
actix:   Downloaded clap v2.33.0
actix:   Downloaded cc v1.0.50
actix:   Downloaded ppv-lite86 v0.2.6
actix:   Downloaded encoding_index_tests v0.1.4
actix:   Downloaded typenum v1.11.2
actix:   Downloaded lru-cache v0.1.2
actix:   Downloaded cmake v0.1.42
actix:   Downloaded content_inspector v0.2.4
actix:   Downloaded console v0.7.7
actix:   Downloaded maybe-uninit v2.0.0
actix:   Downloaded remove_dir_all v0.5.2
actix:   Downloaded crossbeam-utils v0.7.2
actix:   Downloaded derive_builder v0.7.2
actix:   Downloaded syntect v3.2.0
actix:   Downloaded fs_extra v1.1.0
actix:   Downloaded directories v1.0.2
actix:   Downloaded idna v0.2.0
actix:   Downloaded unicase v2.6.0
actix:   Downloaded parking_lot_core v0.7.0
actix:   Downloaded ansi_colours v1.0.1
actix:   Downloaded siphasher v0.3.1
actix:   Downloaded lock_api v0.3.3
actix:   Downloaded shell-words v0.1.0
actix:   Downloaded error-chain v0.12.2
actix:   Downloaded hostname v0.1.5
actix:   Downloaded linked-hash-map v0.5.2
actix:   Downloaded dirs v1.0.5
actix:   Downloaded yaml-rust v0.4.3
actix:   Downloaded walkdir v2.3.1
actix:   Downloaded flate2 v1.0.13
actix:   Downloaded derive_builder_core v0.5.0
actix:   Downloaded darling v0.9.0
actix:   Downloaded version_check v0.9.1
actix:   Downloaded failure_derive v0.1.6
actix:   Downloaded async-trait v0.1.24
actix:   Downloaded termios v0.3.1
actix:   Downloaded plist v0.4.2
actix:   Downloaded bincode v1.2.1
actix:   Downloaded enum-as-inner v0.3.1
actix:   Downloaded onig v4.3.3
actix:   Downloaded unicode-width v0.1.7
actix:   Downloaded semver v0.9.0
actix:   Downloaded strsim v0.8.0
actix:   Downloaded backtrace v0.3.45
actix:   Downloaded textwrap v0.11.0
actix:   Downloaded lazycell v1.2.1
actix:   Downloaded bytestring v0.1.4
actix:   Downloaded socket2 v0.3.11
actix:   Downloaded clicolors-control v1.0.1
actix:   Downloaded vec_map v0.8.1
actix:   Downloaded backtrace-sys v0.1.33
actix:   Downloaded scopeguard v1.1.0
actix:   Downloaded same-file v1.0.6
actix:   Downloaded darling_core v0.9.0
actix:   Downloaded xml-rs v0.8.0
actix:   Downloaded base64 v0.10.1
actix:   Downloaded darling_macro v0.9.0
actix:   Downloaded semver-parser v0.7.0
actix:   Downloaded heck v0.3.1
actix:   Downloaded synstructure v0.12.3
actix:   Downloaded crc32fast v1.2.0
actix:   Downloaded miniz_oxide v0.3.6
actix:   Downloaded onig_sys v69.1.0
actix:   Downloaded line-wrap v0.1.1
actix:   Downloaded strsim v0.7.0
actix:   Downloaded ident_case v1.0.1
actix:   Downloaded rustc-demangle v0.1.16
actix:   Downloaded adler32 v1.0.4
actix:   Downloaded unicode-segmentation v1.6.0
actix:   Downloaded pkg-config v0.3.17
actix:   Downloaded safemem v0.3.3
actix:    Compiling libc v0.2.67
actix:    Compiling proc-macro2 v1.0.9
actix:    Compiling unicode-xid v0.2.0
actix:    Compiling syn v1.0.16
actix:    Compiling cfg-if v0.1.10
actix:    Compiling memchr v2.3.3
actix:    Compiling cc v1.0.50
actix:    Compiling lazy_static v1.4.0
actix:    Compiling smallvec v1.2.0
actix:    Compiling fnv v1.0.6
actix:    Compiling log v0.4.8
actix:    Compiling futures-core v0.3.4
actix:    Compiling slab v0.4.2
actix:    Compiling futures-sink v0.3.4
actix:    Compiling autocfg v1.0.0
actix:    Compiling bytes v0.5.4
actix:    Compiling serde v1.0.104
actix:    Compiling byteorder v1.3.4
actix:    Compiling proc-macro-nested v0.1.3
actix:    Compiling bitflags v1.2.1
actix:    Compiling getrandom v0.1.14
actix:    Compiling futures-task v0.3.4
actix:    Compiling scopeguard v1.1.0
actix:    Compiling arc-swap v0.4.4
actix:    Compiling pin-utils v0.1.0-alpha.4
actix:    Compiling futures-io v0.3.4
actix:    Compiling pin-project-lite v0.1.4
actix:    Compiling matches v0.1.8
actix:    Compiling ppv-lite86 v0.2.6
actix:    Compiling proc-macro2 v0.4.30
actix:    Compiling quick-error v1.2.3
actix:    Compiling itoa v0.4.5
actix:    Compiling regex-syntax v0.6.16
actix:    Compiling rustc-demangle v0.1.16
actix:    Compiling unicode-xid v0.1.0
actix:    Compiling version_check v0.9.1
actix:    Compiling linked-hash-map v0.5.2
actix:    Compiling syn v0.15.44
actix:    Compiling copyless v0.1.4
actix:    Compiling ryu v1.0.2
actix:    Compiling version_check v0.1.5
actix:    Compiling typenum v1.11.2
actix:    Compiling either v1.5.3
actix:    Compiling encoding_index_tests v0.1.4
actix:    Compiling failure_derive v0.1.6
actix:    Compiling percent-encoding v2.1.0
actix:    Compiling unicode-segmentation v1.6.0
actix:    Compiling ident_case v1.0.1
actix:    Compiling strsim v0.7.0
actix:    Compiling pkg-config v0.3.17
actix:    Compiling base64 v0.11.0
actix:    Compiling crc32fast v1.2.0
actix:    Compiling mime v0.3.16
actix:    Compiling safemem v0.3.3
actix:    Compiling same-file v1.0.6
actix:    Compiling adler32 v1.0.4
actix:    Compiling encoding_rs v0.8.22
actix:    Compiling httparse v1.3.4
actix:    Compiling byte-tools v0.3.1
actix:    Compiling unicode-width v0.1.7
actix:    Compiling xml-rs v0.8.0
actix:    Compiling derive_builder v0.7.2
actix:    Compiling subtle v1.0.0
actix:    Compiling dtoa v0.4.5
actix:    Compiling v_htmlescape v0.4.5
actix:    Compiling maybe-uninit v2.0.0
actix:    Compiling fake-simd v0.1.2
actix:    Compiling opaque-debug v0.2.3
actix:    Compiling sha1 v0.6.0
actix:    Compiling ansi_term v0.11.0
actix:    Compiling lazycell v1.2.1
actix:    Compiling semver-parser v0.7.0
actix:    Compiling vec_map v0.8.1
actix:    Compiling strsim v0.8.0
actix:    Compiling fs_extra v1.1.0
actix:    Compiling language-tags v0.2.2
actix:    Compiling fallible-iterator v0.2.0
actix:    Compiling pq-sys v0.4.6
actix:    Compiling humansize v1.1.0
actix:    Compiling siphasher v0.3.1
actix:    Compiling md5 v0.7.0
actix:    Compiling remove_dir_all v0.5.2
actix:    Compiling askama_escape v0.2.0
actix:    Compiling shell-words v0.1.0
actix:    Compiling termcolor v1.1.0
actix:    Compiling percent-encoding v1.0.1
actix:    Compiling thread_local v1.0.1
actix:    Compiling unicode-bidi v0.3.4
actix:    Compiling futures-channel v0.3.4
actix:    Compiling lock_api v0.3.3
actix:    Compiling humantime v1.3.0
actix:    Compiling unicode-normalization v0.1.12
actix:    Compiling encoding-index-japanese v1.20141219.5
actix:    Compiling encoding-index-singlebyte v1.20141219.5
actix:    Compiling encoding-index-simpchinese v1.20141219.5
actix:    Compiling encoding-index-tradchinese v1.20141219.5
actix:    Compiling encoding-index-korean v1.20141219.5
actix:    Compiling lru-cache v0.1.2
actix:    Compiling yaml-rust v0.4.3
actix:    Compiling line-wrap v0.1.1
actix:    Compiling cmake v0.1.42
actix:    Compiling walkdir v2.3.1
actix:    Compiling http v0.2.0
actix:    Compiling bytestring v0.1.4
actix:    Compiling miniz_oxide v0.3.6
actix:    Compiling block-padding v0.1.5
actix:    Compiling textwrap v0.11.0
actix:    Compiling heck v0.3.1
actix:    Compiling semver v0.9.0
actix:    Compiling c2-chacha v0.2.3
actix:    Compiling phf_shared v0.8.0
actix:    Compiling num-traits v0.2.11
actix:    Compiling indexmap v1.3.2
actix:    Compiling num-integer v0.1.42
actix:    Compiling crossbeam-utils v0.7.2
actix:    Compiling nom v4.2.3
actix:    Compiling unicase v2.6.0
actix:    Compiling error-chain v0.12.2
actix:    Compiling phf v0.8.0
actix:    Compiling encoding v0.2.33
actix:    Compiling aho-corasick v0.7.9
actix:    Compiling content_inspector v0.2.4
actix:    Compiling base64 v0.10.1
actix:    Compiling fxhash v0.2.1
actix:    Compiling idna v0.2.0
actix:    Compiling stringprep v0.1.2
actix:    Compiling idna v0.1.5
actix:    Compiling quote v0.6.13
actix:    Compiling quote v1.0.3
actix:    Compiling iovec v0.1.4
actix:    Compiling net2 v0.2.33
actix:    Compiling parking_lot_core v0.7.0
actix:    Compiling signal-hook-registry v1.2.0
actix:    Compiling num_cpus v1.12.0
actix:    Compiling socket2 v0.3.11
actix:    Compiling hostname v0.1.5
actix:    Compiling atty v0.2.14
actix:    Compiling time v0.1.42
actix:    Compiling flate2 v1.0.13
actix:    Compiling clicolors-control v1.0.1
actix:    Compiling termios v0.3.1
actix:    Compiling dirs v1.0.5
actix:    Compiling directories v1.0.2
actix:    Compiling generic-array v0.12.3
actix:    Compiling mime_guess v2.0.3
actix:    Compiling crossbeam-channel v0.4.2
actix:    Compiling threadpool v1.7.1
actix:    Compiling mio v0.6.21
actix:    Compiling url v2.1.1
actix:    Compiling rand_core v0.5.1
actix:    Compiling parking_lot v0.10.0
actix:    Compiling url v1.7.2
actix:    Compiling resolv-conf v0.6.2
actix:    Compiling regex v1.3.4
actix:    Compiling clap v2.33.0
actix:    Compiling digest v0.8.1
actix:    Compiling block-buffer v0.7.3
actix:    Compiling crypto-mac v0.7.0
actix:    Compiling rand_chacha v0.2.1
actix:    Compiling rand_pcg v0.2.1
actix:    Compiling sha2 v0.8.1
actix:    Compiling hmac v0.7.1
actix:    Compiling rand v0.7.3
actix:    Compiling chrono v0.4.10
actix:    Compiling mio-uds v0.6.7
actix:    Compiling tokio v0.2.6
actix:    Compiling console v0.7.7
actix:    Compiling toolchain_find v0.1.4
actix:    Compiling env_logger v0.7.1
actix:    Compiling backtrace-sys v0.1.33
actix:    Compiling onig_sys v69.1.0
actix:    Compiling ansi_colours v1.0.1
actix:    Compiling snmalloc-sys v0.2.5
actix:    Compiling jemalloc-sys v0.3.2
actix:    Compiling postgres-protocol v0.5.0 (https://github.com/fafhrd91/rust-postgres.git#a3be57c0)
actix:    Compiling tempfile v3.1.0
actix:    Compiling postgres-types v0.1.0 (https://github.com/fafhrd91/rust-postgres.git#a3be57c0)
actix:    Compiling darling_core v0.9.0
actix:    Compiling backtrace v0.3.45
actix:    Compiling tokio-util v0.2.0
actix:    Compiling actix-codec v0.2.0
actix:    Compiling snmalloc-rs v0.2.5
actix:    Compiling synstructure v0.12.3
actix:    Compiling v_eval v0.1.0
actix:    Compiling onig v4.3.3
actix:    Compiling jemallocator v0.3.2
actix:    Compiling darling_macro v0.9.0
actix:    Compiling serde_derive v1.0.104
actix:    Compiling proc-macro-hack v0.5.11
actix:    Compiling pin-project-internal v0.4.8
actix:    Compiling derive_more v0.99.3
actix:    Compiling actix-macros v0.1.1
actix:    Compiling enum-as-inner v0.3.1
actix:    Compiling async-trait v0.1.24
actix:    Compiling v_escape_derive v0.5.6
actix:    Compiling actix-web-codegen v0.2.1
actix:    Compiling actix_derive v0.5.0
actix:    Compiling diesel_derives v1.4.1
actix:    Compiling markup-proc-macro v0.4.1
actix: error[E0433]: failed to resolve: could not find `__rt` in `quote`
actix:    --> /usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/failure_derive-0.1.6/src/lib.rs:107:70
actix:     |
actix: 107 | fn display_body(s: &synstructure::Structure) -> Result<Option<quote::__rt::TokenStream>, Error> {
actix:     |                                                                      ^^^^ could not find `__rt` in `quote`
actix: error: aborting due to previous error
actix: For more information about this error, try `rustc --explain E0433`.
actix: error: could not compile `failure_derive`.
actix: warning: build failed, waiting for other jobs to finish...
actix: error: build failed
actix: Removing intermediate container 7a59ded83af5
actix: Docker build failed; terminating
actix: Traceback (most recent call last):
actix:   File "/FrameworkBenchmarks/toolset/utils/docker_helper.py", line 57, in __build
actix:     raise Exception(token['errorDetail']['message'])
actix: Exception: The command '/bin/sh -c RUSTFLAGS="-C target-cpu=native" cargo build --release' returned a non-zero code: 101
actix: Build time: 2m 16s
```

**Comment by JohnTitor** [comment]
It's already fixed in failure_derive 0.1.7.

**Comment by cdbattags** [comment]
@JohnTitor any chance you can point me to the specific module in `actix-web` or `actix-net` that uses `failure_derive` as a dep? I'd like to be sure it'll be updated to 1.7

**Comment by JohnTitor** [comment]
@cdbattags You should see the lockfile. We depend on failure that depends on failure_derive. It's a very known issue, quote 1.0.3 hides private-ish API that failure_derive 0.1.6 depends on. And note that the version that has the fix is not **1.7** but **0.1.7**. There is nothing to do on our side, you have to just update dependencies.

**Comment by cdbattags** [comment]
Thank you for the insights!

Looks like the most recent runs are working fine. We should be all set!
